mappings/net/minecraft/data/client/model/BlockStateModelGenerator.mapping

```diff
@@ -1,12 +1,422 @@
+	METHOD method_25519 registerHopper ()V
+	METHOD method_25520 registerIronBars ()V
+	METHOD method_25521 registerLever ()V
+	METHOD method_25522 registerLilyPad ()V
+	METHOD method_25523 registerNetherPortal ()V
+	METHOD method_25524 registerNetherrack ()V
+	METHOD method_25525 registerObserver ()V
+	METHOD method_25526 registerPistons ()V
+	METHOD method_25527 registerPistonHead ()V
+	METHOD method_25528 registerScaffolding ()V
+	METHOD method_25529 registerRedstoneLamp ()V
+	METHOD method_25530 registerRedstoneTorch ()V
+	METHOD method_25531 registerRepeater ()V
+	METHOD method_25532 registerSeaPickle ()V
+	METHOD method_25533 registerSnows ()V
+	METHOD method_25534 register ()V
+	METHOD method_25535 getBambooBlockStateVariants (I)Ljava/util/List;
+		ARG 1 age
+	METHOD method_25536 getTurtleEggModel (ILjava/lang/String;Lnet/minecraft/class_4944;)Lnet/minecraft/class_2960;
+		ARG 1 eggs
+		ARG 2 prefix
+		ARG 3 texture
+	METHOD method_25537 registerItemModel (Lnet/minecraft/class_1792;)V
+		ARG 1 item
+	METHOD method_25538 registerParentedItemModel (Lnet/minecraft/class_1792;Lnet/minecraft/class_2960;)V
+		ARG 1 item
+		ARG 2 parentModelId
+	METHOD method_25540 excludeFromSimpleItemModelGeneration (Lnet/minecraft/class_2248;)V
+		ARG 1 block
+	METHOD method_25542 registerBuiltinWithParticle (Lnet/minecraft/class_2248;Lnet/minecraft/class_1792;)V
+		ARG 1 block
+		ARG 2 particleSource
 	METHOD method_25543 registerStateWithModelReference (Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;)V
 		ARG 1 block
 		ARG 2 modelReference
+	METHOD method_25544 registerCoral (Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;)V
+		ARG 1 coral
+		ARG 2 deadCoral
+		ARG 3 coralBlock
+		ARG 4 deadCoralBlock
+		ARG 5 coralFan
+		ARG 6 deadCoralFan
+		ARG 7 coralWallFan
+		ARG 8 deadCoralWallFan
+	METHOD method_25545 registerFlowerPotPlant (Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;Lnet/minecraft/class_4910$class_4913;)V
+		ARG 1 plantBlock
+		ARG 2 flowerPotBlock
+		ARG 3 tintType
+	METHOD method_25546 registerCubeWithCustomTexture (Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;Ljava/util/function/BiFunction;)V
+		ARG 1 block
+		ARG 2 otherTextureSource
+		ARG 3 textureFactory
+	METHOD method_25547 registerCrop (Lnet/minecraft/class_2248;Lnet/minecraft/class_2769;[I)V
+		ARG 1 crop
+		ARG 2 ageProperty
+		ARG 3 ageTextureIndices
+	METHOD method_25548 registerTintableCross (Lnet/minecraft/class_2248;Lnet/minecraft/class_4910$class_4913;)V
+		ARG 1 block
+		ARG 2 tintType
+	METHOD method_25549 registerTintableCross (Lnet/minecraft/class_2248;Lnet/minecraft/class_4910$class_4913;Lnet/minecraft/class_4944;)V
+		ARG 1 block
+		ARG 2 tintType
+		ARG 3 texture
+	METHOD method_25550 registerNorthDefaultHorizontalRotatable (Lnet/minecraft/class_2248;Lnet/minecraft/class_4944;)V
```

Comment:
I think `registerHorizontalRotatable`, and adding the fact that north is the default in a javadoc, is better.